projects/productconfig/.release-it.json

```diff
@@ -0,0 +1,34 @@
+{
+  "git": {
+    "requireCleanWorkingDir": true,
+    "requireUpstream": false,
+    "tagName": "productconfig-${version}",
+    "commitMessage": "Bumping productconfig version to ${version}",
+    "tagAnnotation": "Bumping productconfig version to ${version}"
+  },
+  "npm": {
+    "publishPath": "./../../dist/productconfig"
+  },
+  "hooks": {
+    "after:version:bump": "cd ../.. && yarn build:core:lib:productconfig"
+  },
+  "github": {
+    "release": true,
+    "assets": ["../../docs.tar.gz", "../../docs.zip"],
+    "releaseName": "@spartacus/productconfig@${version}",
+    "releaseNotes": "cd ../.. && npm run --silent release-notes:productconfig -- --to productconfig-${version}"
```

Comment:
Make sure to define `release-notes:productconfig` in `package.json` (see `release-notes:cds` as an example).

Reply:
I think we have changed that a week ago. We now just execute script from here, we skip the npm script part.